src/chrome/content/rules/RCI.com.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+	Non-functional hosts
+		Timeout was reached:
+		- vtrends.rci.com
+
+		SSL peer certificate was not OK:
+		- tktermal.rci.com
+
+		Peer certificate cannot be authenticated with given CA certificates:
+		- pgs.rci.com
+
+		Incomplete certificate chain error:
+		- rcieliterewards.rci.com
+-->
+<ruleset name="RCI.com (partial)">
+	<target host="rci.com" />
+	<target host="www.rci.com" />
+	<target host="ai.rci.com" />
+	<target host="api.rci.com" />
+	<target host="arrivalguides.rci.com" />
+	<target host="b2b.rci.com" />
+	<target host="discover.rci.com" />
+	<target host="m.rci.com" />
+	<exclusion pattern="^http://m\.rci\.com/$" />
+		<test url="http://m.rci.com/rcimobile/" />
```

Comment:
I think `$` can be redirected to `/rcimobile`. Please also add document the issue.